tests/tom-swifty-test/TestRunning.cs

```diff
@@ -686,7 +686,14 @@ public static string Execute (string workingDirectory, string executable, Platfo
 					var output = new StringBuilder ();
 					var responseFile = Path.Combine (workingDirectory, name + ".rsp");
 					File.WriteAllText (responseFile, mmp.ToString ());
-					var rv = ExecAndCollect.RunCommand ("/Library/Frameworks/Xamarin.Mac.framework/Versions/Current/bin/mmp", "--link_flags=-headerpad_max_install_names " + StringUtils.Quote ($"@{responseFile}"), output: output, verbose: true);
+					// The test environment is coming is with PKG_CONFIG_LIBDIR=""
```

Comment:
do we know who is setting that? We could remove it

Reply:
No, no we don't.